docs/template-helpRequest.md

```diff
@@ -0,0 +1,66 @@
+**Help Request**
+
+**Issue Title:** Help Request: [summary of your problem]
+
+
+* **Operating System**
+
+
+* **JDK Version**
+
+
+* **Eclipse Version**
+
+
+* **Firefox / Chrome Version (where applicable)**
```

Comment:
You should not need the empty lines between the bullet points, and add a ":" at the end of each point. Something like:
```
* **A:**
* **B:**
```